backend/backend/version.json

```diff
@@ -0,0 +1,3 @@
+{
+  "version": "4.2.1"
```

Comment:
what's this

Reply:
dunno

Reply:
It has something to do with ffmeg I believe. I will add it to gitIgnore